README.md

```diff
@@ -66,31 +66,74 @@ JSONObject result = CoreApi.chargeTransaction(param);
 // SnapApi request with global config
 JSONObject result = SnapApi.createTransaction(param);
 ```
-
-All the request can accept an optional Config object. This is used if you want to set an others' config like idempotency key, proxy,
-or if you want to pass the server-key on each method, or use multiple account on each method.
+#### Per-request Configuration
+All the request can accept an optional Config object. This is used if you want to set an others' config like idempotency key, proxy, override/append notification url
+or want to use multiple account on each method. Please follow the steps given below.
```

Comment:
Wording is a bit weird. Fix to:
> It is also possible if on each of individual requests you want to set a unique/different configuration; like idempotency key, proxy, override/append notification url, or multiple account API keys. Please follow the steps given below.

Reply:
Also which method is "thread-safe for multi-thread concurrency usage" is unclear. Can you explicitly state that? so that it is more visible.

Reply:
Done @rizdaprasetya please kindly review ya